sc-poc/managers/configman/common/http.go

```diff
@@ -87,59 +89,58 @@ func getAPIRoutes() caddyhttp.Route {
 	}
 }
 
-func getConfigRoutes() caddyhttp.Route {
+func getConfigRoutes(config ConfigType) caddyhttp.Route {
 	// Make route list for the sub router
-	routeList := caddyhttp.RouteList{
-		// Open API for the config and operation endpoints
-		caddyhttp.Route{
-			Group:          "openapi",
-			MatcherSetsRaw: utils.GetCaddyMatcherSet([]string{"/v1/config/openapi.json"}, []string{http.MethodGet}),
-			HandlersRaw:    utils.GetCaddyHandler("config_openapi", nil),
-		},
-
-		// Admin auth middleware
-		caddyhttp.Route{
-			Group:       "auth",
-			HandlersRaw: utils.GetCaddyHandler("admin_auth", nil),
-		},
-
-		// Config routes
-		caddyhttp.Route{
-			Group:          "config",
-			MatcherSetsRaw: utils.GetCaddyMatcherSet([]string{"/v1/config/*"}, []string{http.MethodGet}),
-			HandlersRaw:    utils.GetCaddyHandler("config_get", nil),
-		},
-		caddyhttp.Route{
-			Group:          "config",
-			MatcherSetsRaw: utils.GetCaddyMatcherSet([]string{"/v1/config/*"}, []string{http.MethodDelete}),
-			HandlersRaw:    utils.GetCaddyHandler("config_delete", nil),
-		},
-		caddyhttp.Route{
-			Group:          "config",
-			MatcherSetsRaw: utils.GetCaddyMatcherSet([]string{"/v1/config/*"}, []string{http.MethodPost}),
-			HandlersRaw:    utils.GetCaddyHandler("config_apply", nil),
-		},
-
-		// Operation routes
-		caddyhttp.Route{
-			Group:          "operation",
-			MatcherSetsRaw: utils.GetCaddyMatcherSet([]string{"/v1/operation/*"}, []string{}),
-			HandlersRaw:    utils.GetCaddyHandler("operation", nil),
-		},
+	configRoutes := caddyhttp.RouteList{}
+	for k := range config {
```

Comment:
We should add a `/v1/config/openapi.json` route for returning the openapi doc for config changes as well. Make this as a seperate issue that you can fix in a separate PR

Reply:
I didn't understand this

Reply:
We already have a route for returning the openapi doc